client/src/utils/formatDateTime.js

```diff
@@ -0,0 +1,13 @@
+// Function that converts date and time input into ISO format
+export const formatDateTime = (date, time) => {
```

Comment:
moment.js might be a better solution than formatting it yourself

Reply:
I have incorporated moment.js to format the dates prior to sending to the database. Timezone differences will be dealt with on retrieving request from database rather than when creating request.